### ISSUE TITLE
Can we use pretrain model (mace_mp) and retrain with our configurations?

### ISSUE THREAD

**Issue by Amitcuhp** [ACEsuit/mace#308]
I am curious to know if we can use mace_mp model and start from the epoch given and retrain with new configurations?

Is it possible?

!

**Comment by davkovacs** [comment]
Yes, it is implemented in the foundations branch: https://github.com/ACEsuit/mace/tree/foundations

you can select "small", "medium" and "large" models and it will automatically select the right hypers, no need to specify them. Here is an example for finetuning the small model:

mace_run_train ^
    --name="MACE" ^
    --foundation_model="small" ^
    --train_file="D:\Work\mace_mp\mp_traj_combined_1p.xyz" ^
    --valid_fraction=0.01 ^
    --test_file="D:\Work\mace_mp\mp_traj_combined_1p.xyz" ^
    --energy_weight=1.0 ^
    --forces_weight=1.0 ^
    --E0s="average" ^
    --lr=0.01 ^
    --error_table="PerAtomMAE" ^
    --scaling="rms_forces_scaling" ^
    --batch_size=2 ^
    --max_num_epochs=6 ^
    --ema ^
    --ema_decay=0.99 ^
    --amsgrad ^
    --default_dtype="float32"^
    --device=cuda ^
    --seed=3

**Comment by Amitcuhp** [comment]
That's wonderfull 
I have also used universal branch in which the loss function is universal.

Thank you for the reply
